R/teal_slice.R

```diff
@@ -338,6 +341,42 @@ is.teal_slices <- function(x) { # nolint
 }
 
 
+# utils -----------------------------------------------------------------------------------------------------------
+
+trim_character <- function(x, nchars, min = 0) {
```

Comment:
@chlebowa I think format only has `show_all` and `nchars` - not that bad. It's the `trim_character` that is getting `worse` if a function with 3 lines can be called complicated. I'm fine with nchars being set to 20, but this parameter needs to be present anyway so it can also be called with `nchars = NULL` for `slices_store`

Reply:
Let's talk it over on a call.

Reply:
fixed by chlebowa in next commits